languages/csharp/Bitwarden.Sdk/BitwardenLibrary.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Bitwarden.Sdk;
+
+internal static class BitwardenLibrary
+{
+    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
+    private static extern BitwardenSafeHandle init(string settings);
+
+    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
+    private static extern void free_mem(IntPtr handle);
+
+    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
+    private static extern string run_command(string json, BitwardenSafeHandle handle);
+
+    internal static BitwardenSafeHandle Init(string settings)
+    {
+        return init(settings);
+    }
+
+    internal static void FreeMemory(IntPtr handle)
```

Comment:
I had issues using BitwardenSafeHandle here so I replaced it with IntPtr. Problem was that SafeHandle would close the handle and it will fail here because  handle is closed. (In debug I would see that in ReleaseHandle(), before calling BitwardenLibrary.FreeMemory(handle); handle is closed)
I was looking at [this](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.safehandle?view=net-6.0&redirectedfrom=MSDN) example and implemented the same way.